docs/apps/crypto-trading.md

```diff
@@ -3,178 +3,197 @@ sidebar_position: 2
 title: Crypto Trading Bot
 ---
 
-## On GDN
+## Crypto App details
 
 | **Tenant** | **Fabric** | **Stream App** | **GUI** | **Source Code**|
 |----------- |----------|----|-----------|-----------|
 |  `demo@macrometa.io` | `_system` | `crypto-trading-app` | [**Crypto Trading GUI**](https://macrometacorp.github.io/tutorial-cryptotrading/) |[github](https://github.com/Macrometacorp/tutorial-cryptotrading)|
 
-## Stream App
+Read our [Blog Post](https://www.macrometa.com/blog/cryptocurrency-trading-building-a-multi-exchange-global-trading-bot) about exchange arbitrage and how the app works.
+
+## Configure the Crypto App
+
+1. Log in to the [Macrometa Console](https://auth.paas.macrometa.io/) 
+2. Go to the Collections tab and create a new `Doc` collection named `trades` with `collection stream enabled` checked.
+3. Click **Stream Workers**.
+4. Click **New Stream Worker** named `Crypto-Trading-App`.
+5. Copy and past the [code example](#crypto-app-stream-worker-code) into the editor.
+6. Click **Save**. When you do we validate your code.
```

Comment:
```suggestion
6. Click **Save**. 

    Macrometa validates your code.
```

Reply:
Can you add a little more here about the validation? What if it says the code is invalid? Is the validate line even useful in this context, or should we take it out?